docs/input_output.md

```diff
@@ -0,0 +1,196 @@
+# File handling with dsub
+
+dsub is designed to make it easy to get input files from a remote storage
```

Comment:
An on-premises job scheduler like Grid Engine typically uses a shared file system. Your scripts can reference files by their absolute paths. With dsub, you'll use absolute paths to cloud storage buckets instead of a shared file system. 

Behind the scenes, dsub will copy input files from bucket paths to local disk, and copy outputs from local disk back to bucket paths.